sdk/ai/Azure.AI.Agents.Persistent/samples/Sample21_PersistentAgents_Sharepoint.md

```diff
@@ -0,0 +1,184 @@
+# Sample for use of an agent with SharePoint in Azure.AI.Agents.Persistent.
+
+To enable your Agent to perform actions using SharePoint, use `SharepointToolDefinition` along with a connection.
+
+1. First, create an agent client and read the environment variables, which will be used in the next steps.
+
+```C# Snippet:AgentsSharepoint_CreateProject
+var projectEndpoint = System.Environment.GetEnvironmentVariable("PROJECT_ENDPOINT");
+var modelDeploymentName = System.Environment.GetEnvironmentVariable("MODEL_DEPLOYMENT_NAME");
+var connectionId = System.Environment.GetEnvironmentVariable("AZURE_SHAREPOINT_CONNECTION_ID");
```

Comment:
Should we not show how to get this from Project SDK connection operations?

Reply:
There are multiple samples where that approach would be more customer-friendly. Let me address that as a separate task.